### PR TITLE
[BEE-27761] Updated based on editorial review comments

### PR DESCRIPTION
**Pull request by jsmith-cb**
During the editorial review process, there were several issues identified with the plugin.

### CODE REVIEW

**Review by karakurty — APPROVED** [review]

File: help/help.adoc [hunk 1]
```diff
@@ -16,67 +16,67 @@ IMPORTANT: For all parameter descriptions below, required parameters are shown i
...
-image:cloudbees-common::cd-plugins/ec-dsldeploy/help/installdsl.png[installDsl parameters]
+image:cloudbees-common-sda::cd-plugins/ec-dsldeploy/help/installdsl.png[installDsl parameters]
 
 === InstallDslFromDirectory
 
-Deploy a DSL file structure from a directory on your {CD} server.
+Deploy a DSL file structure from a directory on your {PRODUCT} server.
 
-This procedure main usage is to set up a CI process. It will extract your code from your SCM in a local directory on your Dev {CD} server, and deploy it from there.
+This procedure main usage is to set up a CI process. It will extract your code from your SCM in a local directory on your `Dev` {PRODUCT} server, and deploy it from there.
```
Comment (b-fisher-cb) [inline help/help.adoc:43]:
```suggestion
This procedure's main usage is to set up a CI process. It extracts your code from your SCM in a local directory on your `Dev` {PRODUCT} server, and deploys it.
```

File: help/help.adoc [hunk 2]
```diff
@@ -16,67 +16,67 @@ IMPORTANT: For all parameter descriptions below, required parameters are shown i
...
+[cols="1a,1a",options="header",]
 |===
 |Parameter |Description
-|Directory Path |The location of the DSL code on your Dev {CD} server.
-|Server Pool or Resource |The name of the pool or resource where the artifact will be retrieved. Use "." if you have retrieved in your workspace.
+|Directory Path |The location of the DSL code on your `Dev` {PRODUCT} server.
+|Server Pool or Resource |The name of the pool or resource where the artifact will be retrieved. Use `.` if you have retrieved in your workspace.
 |Overwrite mode |Select to enable overwrite mode. If the object exists, all attributes and child objects are overwritten by the values in the DSL files.
-|Additional argument(s) for evalDsl call. ie: --timeout or --debug. Values used in --timeout will be used as timeouts for ectool, evalDsl, and the job step time out used to import the DSL.
+|Additional argument(s)| For evalDsl calls, additional arguments such as  `--timeout` or `--debug`. Values used in `--timeout` will be used as timeouts for ectool, evalDsl, and the job step time out used to import the DSL.
```
Comment (b-fisher-cb) [inline help/help.adoc:51]:
```suggestion
|Additional argument(s)| For evalDsl calls, additional arguments such as  `--timeout` or `--debug`. Values used in `--timeout` are used as timeouts for ectool, evalDsl, and the job step time out used to import the DSL.
```

File: help/help.adoc [hunk 3]
```diff
@@ -16,67 +16,67 @@ IMPORTANT: For all parameter descriptions below, required parameters are shown i
...
+[cols="1a,1a",options="header",]
 |===
 |Parameter |Description
-|Project Name |Name of the project to instal.
-|Project directory |The location of the project DSL code on your Dev CloudBeed CD server.
+|Project Name |Name of the project to install.
+|Project directory |The location of the project DSL code on your `Dev` {PRODUCT} server.
 |Overwrite mode |Select to enable overwrite mode. If the object exists, all attributes and child objects are overwritten by the values in the DSL files.
-|Additional argument(s) for evalDsl call. ie: --timeout or --debug. Values used in --timeout will be used as timeouts for ectool, evalDsl, and the job step time out used to import the DSL.
+|Additional argument(s)| For evalDsl calls, additional arguments such as  `--timeout` or `--debug`. Values used in `--timeout` will be used as timeouts for ectool, evalDsl, and the job step time out used to import the DSL.
```
Comment (b-fisher-cb) [inline help/help.adoc:70]:
```suggestion
|Additional argument(s)| For evalDsl calls, additional arguments such as  `--timeout` or `--debug`. Values used in `--timeout` are used as timeouts for ectool, evalDsl, and the job step time out used to import the DSL.
```

File: help/help.adoc [hunk 4]
```diff
@@ -87,48 +87,50 @@ Export DSL for a top level {CD} object, with options for which child objects and
...
 
 This procedure main usage is to set up a CI process.
 
-[cols=",",options="header",]
+[cols="1a,1a",options="header",]
 |===
 |Parameter |Description
-|Server Resource |The resource where the DSL files will be checked out from git and imported to the {CD} server.
-|Destination Directory |The directory on the resource where the source tree will be created and from where the DSL files are read to be imported in {CD} server.
+|Server Resource |The resource where the DSL files will be checked out from git and imported to the {PRODUCT} server.
```
Comment (b-fisher-cb) [inline help/help.adoc:107]:
```suggestion
|Server Resource |The resource where the DSL files are checked out from Git and imported to the {PRODUCT} server.
```

File: help/help.adoc [hunk 5]
```diff
@@ -87,48 +87,50 @@ Export DSL for a top level {CD} object, with options for which child objects and
...
+|Destination Directory |The directory on the resource where the source tree will be created and from where the DSL files are read to be imported in {PRODUCT} server.
```
Comment (b-fisher-cb) [inline help/help.adoc:108]:
```suggestion
|Destination Directory |The directory on the resource where the source tree is created and from where the DSL files are read to be imported in {PRODUCT} server.
```

File: help/help.adoc [hunk 6]
```diff
@@ -87,48 +87,50 @@ Export DSL for a top level {CD} object, with options for which child objects and
...
 |Cleanup? |This option will delete the destination directory with the source tree after the job execution.
 |Overwrite mode |Select to enable overwrite mode. If the object exists, all attributes and child objects are overwritten by the values in the DSL files.
 |Configuration |The name of a saved SCM configuration.
-|Git repository |URL to the repository to pull from. ie: 'git://server/repo.git'.
-|Commit Hash |The Commit Hash to update the index. Note: This will move the HEAD.
-|Remote Branch |The name of the Git branch to use. ie: 'experimental'.
+|Git repository |The URL to the repository to pull from, for example `git://server/repo.git`.
+|Commit Hash |The Commit Hash to update the index.
+
+NOTE: This will move the HEAD.
```
Comment (b-fisher-cb) [inline help/help.adoc:115]:
```suggestion
NOTE: This moves the HEAD.
```

**Review by b-fisher-cb — APPROVED** [review]
lgtm. Only a few small suggestions based on https://docs.cloudbees.com/docs/beewrite/latest/best-practices#_avoid_future_tense.

**Pull request merged**